v20.1/admin-ui-cluster-overview-page.md

```diff
@@ -3,88 +3,112 @@ title: Cluster Overview Page
+## Cluster Overview panel
 
-The **Cluster Overview** panel provides the following metrics:
+Use the **Cluster Overview** panel to quickly assess the capacity and health of your cluster.
+
+<img src="{{ 'images/v20.1/admin-ui-cluster-overview-panel.png' | relative_url }}" alt="CockroachDB Admin UI cluster overview" style="border:1px solid #eee;max-width:100%" />
 
 Metric | Description
 --------|----
-Capacity Usage | <ul><li>Used capacity: The storage capacity used by CockroachDB (represented as a percentage of total storage capacity allocated across all nodes).</li><li>Usable capacity: The space available for CockroachDB data storage (i.e., the storage capacity of the machine excluding the capacity used by the Cockroach binary, operating system, and other system files). </li></ul>
-Node Status | <ul><li>The number of [live nodes](#live-nodes) in the cluster.</li><li>The number of suspect nodes in the cluster. A node is considered suspect if its liveness status is unavailable or the node is in the process of decommissioning.</li><li>The number of [dead nodes](#dead-nodes) in the cluster.</li>
+Capacity Usage | <ul><li>Used capacity: The total disk space in use by CockroachDB data across all nodes. This is also shown as a percentage of the total usable capacity across all nodes, and excludes the disk space used by the Cockroach binary, operating system, and other system files.</li><li>Usable capacity: The total disk space reserved for CockroachDB data. This is defined by [`--store`](cockroach-start.html#store) and is distinct from the total disk size of your machines.</li></ul>
+Node Status | <ul><li>The number of `LIVE` nodes in the cluster.</li><li>The number of `SUSPECT` nodes in the cluster. A node is considered suspect if its [liveness status is unavailable](cluster-setup-troubleshooting.html#node-liveness-issues) or the node is in the process of [decommissioning](#decommissioned-nodes).</li><li>The number of `DEAD` nodes in the cluster.</li>
 Replication Status | <ul><li>The total number of [ranges](architecture/overview.html#glossary) in the cluster.</li><li>The number of [under-replicated ranges](admin-ui-replication-dashboard.html#review-of-cockroachdb-terminology) in the cluster. A non-zero number indicates an unstable cluster.</li><li>The number of [unavailable ranges](admin-ui-replication-dashboard.html#review-of-cockroachdb-terminology) in the cluster. A non-zero number indicates an unstable cluster.</li>
 
 ## Node List
 
-The **Node List** is the default view on the **Overview** page.
-<img src="{{ 'images/v20.1/admin-ui-node-list.png' | relative_url }}" alt="CockroachDB Admin UI" style="border:1px solid #eee;max-width:100%" />
+The **Node List** groups nodes by locality. The highest-level locality tier is used to organize the Node List. Hover over a locality to see all localities for the group of nodes.
```

Comment:
@Annebirzin I left a question for you about this in Figma.

Reply:
Never mind! Found the answer in https://github.com/cockroachdb/cockroach/pull/45041